plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryConfig.java

```diff
@@ -49,6 +49,7 @@ public void testDefaults()
                 .setViewsEnabled(false)
                 .setArrowSerializationEnabled(false)
                 .setQueryResultsCacheEnabled(false)
+                .setQueryAllowLargeResults(false)
```

Comment:
> I don't want to explicitly add a test for it as it will cost too much to run it on the CI.

How about adding a test with a small dataset? Even if the data is smaller than the limitation, better than nothing in my opinion.